lizard_auth_client/tests.py

```diff
@@ -28,13 +27,22 @@
 import json
 import jwt
 import logging
-import mock
+from unittest import mock
 import pprint
-import uuid
 
 logger = logging.getLogger(__name__)
 fake = Faker()
 
+SSO_SERVER_URLS = {
```

Comment:
The unittests were actually running GET requests against the staging SSO server. This sporadically went wrong, so I hardcoded the response here.